src/main/java/EncryptionAlgorithms/DoubleEncryption.java

```diff
@@ -14,17 +15,17 @@ public DoubleEncryption(EncryptionAlgorithm firstAlgorithm, EncryptionAlgorithm
         super("DoubleEncryption");
         this.algorithm1 = firstAlgorithm;
         this.algorithm2 = secondAlgorithm;
-        this.secondKey = (new Key("DoubleEncryption")).getKey();
+        this.secondKey = (new Key()).getKey();
     }
 
     @Override
-    public String encryptFile(String stringToEncrypt, int encryptionKey) throws IOException {
+    public String encryptFile(String stringToEncrypt, int encryptionKey) throws IOException, NoSuchElementException, ClassCastException {
         String middleEncryption = algorithm1.encryptFile(stringToEncrypt, encryptionKey);
         return algorithm2.encryptFile(middleEncryption, secondKey);
     }
 
     @Override
-    public String decryptFile(String stringToDecrypt, int decryptionKey) throws IOException {
+    public String decryptFile(String stringToDecrypt, int decryptionKey) throws IOException, NoSuchElementException, ClassCastException {
```

Comment:
you not decrypt file here - change function name to just decrypt or decryptString etc.
*** in All Algorithems ***